index.html

```diff
@@ -0,0 +1,120 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Joel Atugonza (Billions)</title>
+  <link rel="stylesheet" href="styles/mobile.css">
+  <link rel="stylesheet" href="styles/style.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;300;400;500;700;900&display=swap" rel="stylesheet">
+</head>
+
+<body>
+  <nav>
+    <div class="nav-container">
+      <div class="name">Atugonza Joel (Billions)</div>
+      <div class="mobile-menu-icon">
+        <svg xmlns="http://www.w3.org/2000/svg" width="2.5rem" height="2.5rem" fill="none" viewBox="0 0 24 24" stroke="currentColor">
+          <path stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M4 6h16M4 12h16M4 18h16" />
+        </svg>
+      </div>
+    </div>
+  </nav>
+  <section id="hero-section">
+    <div class="hero-section-container">
+      <div class="heading">
+        <div class="title">Atugonza Joel</div>
+        <div class="sub-title">Software Developer</div>
+      </div>
+      <div class="description">
+        <p>
+          I can help you build a product , feature or website Look through some of my work and experience!
+          If you like what you see and have a project you need coded, don’t hestiate to contact me.
+        </p>
+      </div>
+    </div>
+  </section>
+
+  <div class="devider">
+  </div>
+
+  <section id="skills">
+    <div class="skills-container">
+      <div class="skill">
+        <div class="skill-title">Languages</div>
+        <div class="skill--info">
+          <ul>
+            <li>Javascript</li>
+            <li>Ruby</li>
+            <li>Html</li>
+            <li>Css</li>
+          </ul>
+        </div>
+      </div>
+      <div class="skill">
+        <div class="skill-title">Frameworks</div>
+        <div class="skill--info">
+          <ul>
+            <li>Boostrap</li>
+            <li>Ruby on Rails</li>
+            <li>Laravel</li>
+            <li>React</li>
+          </ul>
+        </div>
+      </div>
+      <div class="skill">
+        <div class="skill-title">Skills</div>
+        <div class="skill--info">
+          <ul>
+            <li>Github</li>
+            <li>Codepen</li>
+            <li>Gitlab</li>
+            <li>Terminal</li>
+          </ul>
+        </div>
+      </div>
+    </div>
+  </section>
+
+  <div class="devider">
+  </div>
+
+  <section id="socials">
+    <div class="social-container">
+      <div class="social-icon">
```

Comment:
- The list of icons uses a ``ul`` tag to group them, and ``li`` per icon; each icon has ``a`` link ( ``a`` tag)

Reply:
I already did this perfectly.

Reply:
Done